.github/workflows/ci.yaml

```diff
@@ -0,0 +1,61 @@
+on:
+  push:
+    branches:
+      - master
+      - release/*
+  pull_request:
+jobs:
+  unit-testing:
+    name: PHPUnit (PHP ${{ matrix.php-versions }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4', '8.0', '8.1']
+        coverage: ['pcov']
+        code-analysis: ['no']
+        include:
+          - php-versions: '7.1'
```

Comment:
There is a leftover mention of PHP 7.1

Reply:
```suggestion
          - php-versions: '7.2'
```

Maybe?

Reply:
This add 7.1 to the matrix with different config (no coverage but with code analysis)

Reply:
I can answer that myself! https://github.com/sabre-io/xml/blob/master/composer.json says PHP 7.1 up is supported.